src/__init__.py

```diff
@@ -12600,19 +12605,20 @@ def extractIMGINFO(self, hashes=0):
                     img_size = img.w() * img.h() * img.n()
             cs = mupdf.FzColorspace(mupdf.ll_fz_keep_colorspace(img.m_internal.colorspace))
             block_dict = dict()
-            block_dict[ dictkey_number] = block_n
-            block_dict[ dictkey_bbox] = JM_py_from_rect(block.m_internal.bbox)
-            block_dict[ dictkey_matrix] = JM_py_from_matrix(block.i_transform())
-            block_dict[ dictkey_width] = img.w()
-            block_dict[ dictkey_height] = img.h()
-            block_dict[ dictkey_colorspace] = mupdf.fz_colorspace_n(cs)
-            block_dict[ dictkey_cs_name] = mupdf.fz_colorspace_name(cs)
-            block_dict[ dictkey_xres] = img.xres()
-            block_dict[ dictkey_yres] = img.yres()
-            block_dict[ dictkey_bpc] = img.bpc()
-            block_dict[ dictkey_size] = img_size
+            block_dict[dictkey_number] = block_n
+            block_dict[dictkey_bbox] = JM_py_from_rect(block.m_internal.bbox)
+            block_dict[dictkey_matrix] = JM_py_from_matrix(block.i_transform())
+            block_dict[dictkey_width] = img.w()
+            block_dict[dictkey_height] = img.h()
+            block_dict[dictkey_colorspace] = mupdf.fz_colorspace_n(cs)
+            block_dict[dictkey_cs_name] = mupdf.fz_colorspace_name(cs)
+            block_dict[dictkey_xres] = img.xres()
+            block_dict[dictkey_yres] = img.yres()
+            block_dict[dictkey_bpc] = img.bpc()
+            block_dict[dictkey_size] = img_size
             if hashes:
-                block_dict[ "digest"] = digest
+                block_dict["digest"] = digest
+            block_dict["has-mask"] = has_mask
```

Comment:
Isn't `has-mask` redundant - can't the user simply use `block_dict['mask'] is not None`?

Reply:
I could have used "mask" as key name. Didn't do it to avoid the misconception that this **_is_** the mask. This method in general does not return image binary data at all - only metadata.
So this key name represents an effort to make sure it is information **_about_** the image - not the image itself.

Reply:
but your reaction in itself show that it is a confusing choice. But we definitely should use bool values.